Changes WakuInfo Field (#476)
vacp2p/rfc: content/docs/rfcs/16/README.md

| Field | Type | Inclusion | Description |
| ----: | :---: | :---: |----------- |

> | `listenAddresses` | `Array`[`String`] | mandatory | Addresses that the node is listening for |

Reviewer: ```suggestion
| `listenAddresses` | `Array`[`String`] | mandatory | Listening addresses of the node |
```

Not your wording, but the dangling preposition has been bothering me for a while, so I'm taking advantage of the opportunity 😄
